uint8 traj_type  # 軌道生成タイプ

uint8 TRAJECTORY_NORMAL=0  # 直線移動 (最速)
uint8 TRAJECTORY_EVEN=1  # 均等分割法 (時間で等間隔移動)